Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ParameterWindow.cpp

```diff
@@ -842,23 +841,28 @@ namespace EMStudio
             return;
         }
 
-        ParameterCreateEditDialog* createEditParameterDialog = new ParameterCreateEditDialog(m_plugin, this);
-        createEditParameterDialog->Init();
+        if (!m_parameterCreateEditWidget)
```

Comment:
It maybe a better idea to just always create the ParameterCreateEditorWidget on init. This way you don't have to deal with checking nullptr all the time making the code overcomplicate. It's a rather lite class to create anyway.